assets/data/cards_pt-br.json

```
@@ -296,6 +296,21 @@
       "title": "Pull Request",
       "description": "Trata-se de uma requisição de alteração dentro de um projeto. Essa requisição pode ser feita para corrigir um problema, adicionar uma nova funcionalidade, sugerir alterações, entre outros motivos. Essa requisição de alteração é feita para que haja uma avaliação do que está sendo alterado dentro do projeto, entendendo de faz sentido para o contexto daquele momento. E essa requisição pode ficar pendente até que pessoas responsáveis ou que tenham a permissão, aprovem. Outras regras podem ser adicionadas nessas requisições, como build da aplicação, rodar testes com sucesso e o que mais fizer sentido para o projeto em questão.",
       "tags": ["Conceito"]
-    }    
+    },
+    {
+      "title": "Garbage Collector",
+      "description": "Literalmente \"coletor de lixo\" em inglês, é um tipo de software auxiliar que é executado ao lado de uma aplicação principal, percebendo memória que foi alocada (designada ao programa) mas não é mais necessária, permitindo que seja retornada ao sistema operacional. Geralmente, é usado por determinação conta da linguagem utilizada, como Java, C# e Javascript",
+      "tags": ["Conceito"]
+    },
+    {
+      "title": "Computação em Nuvem",
+      "description": "De modo geral, refere-se ao uso de computação disponível através da internet e abstraído do equipamento físico utilizado. É um conceito extremamente amplo, que compreende desde armazenamento pessoal até aplicações que são executadas na nuvem. O conceito de computação em nuvem contrasta com a ideia de computação on-premise, que se dá quando o usuário também administra o equipamento que disponibiliza um serviço.",
```

Sugiro que possamos fazer uma versão simplificada que evita jargões técnicos e torna o conceito mais acessível.

```suggestion
      "description": "De modo geral, refere-se ao uso de serviços de computador, como armazenamento de arquivos e execução de programas, através da internet. Isso significa que você não precisa de equipamentos físicos, pois tudo é feito online. Você pode acessar seus dados e programas de qualquer lugar, a qualquer momento, sem se preocupar com a manutenção dos equipamentos físicos.",
```